CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Issues
+
+Issues have three types: bug, feature request, and support. When reporting a bug, you must include
+the operating system used, any relevant information about the tech stack, and the feature flags
+used. Feature request issues should answer these questions: 
+
+```
+## What would you like implemented? What do you want that Twilight lacks?
+
+## What use case does this request address? This should be more than a single project's use case.
+
+## Are you willing to help towards contributing this feature?
+
+## Is there any other information that we should know?
+```
+
+Before making a support issue or bug report, be sure to consider joining the
+[Discord](https://discord.gg/7jj8n7D) and bringing up your topic in the #support channel.
+
+# Pull Requests
+
+Pull requests must be named with the format `crate: short description of change`, and should use
+lower case letters. If the change spans more than one crate, separate the crate names with a comma
+and a space: `crate1, crate2: short description of change`. Pull requests must be made from a new
+branch. If a PR is made from the `trunk` branch, it will be denied. If adding a feature or
+enhancement, use the term `add` or something sufficiently similar. If fixing a bug, use the term
+`fix`, or something sufficiently similar. Avoid force-pushing to a pull request branch, as this
+erases review comment history.
```

Comment:
Possibly add a section on when members should merge pull requests. ie. right after the 2 approvals